extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -56,14 +44,12 @@ export class SqlDatabaseTree extends AssessmentDialogComponent {
 		}).component();
 
 		component.addItem(this.createSearchComponent(view), { flex: '0 0 auto' });
-		component.addItem(this.createInstanceComponent(view), { flex: '0 0 auto' });
-		component.addItem(await this.createDatabaseComponent(view), { flex: '1 1 auto' });
+		component.addItem(await this.createInstanceComponent(view), { flex: '0 0 auto' });
+		component.addItem(await this.createDatabaseComponent(view, dbs), { flex: '1 1 auto' });
```

Comment:
I noticed that in these functions you're actually populating the values (for example using azdata.connection.listDatabases)

In general you should avoid blocking the UI from being created while you wait for things to load - instead use the loading components as necessary while you load things in the background. Querying servers can take quite a while and having a blank screen while it's waiting for those to load is not a great user experience.